src/index.js

```diff
@@ -3,13 +3,7 @@
 /* eslint-disable import/no-mutable-exports */
```

Comment:
- [ ] As linters are a requirement to work on Microverse, you shouldn't be able to disable them. Kindly remove these disable rules and fix the errors. This will improve your code skills and your project will look more professional and clean

![image](https://user-images.githubusercontent.com/93630700/207457793-5815dc39-c04b-44ad-81b9-9ee519a20b4d.png)